### ISSUE TITLE
::selection

### ISSUE THREAD

**Issue by diomed** [kognise/water.css#52]
By darkening the background color by 40% we can get a nice background color for selections.

I propose adding something like this:

```scss
::selection {
  background-color: darken($background-color, 40%);
}
```

**Comment by kognise** [comment]
I attempted editing your issue to possibly make it more clear - did I grasp what you meant to say correctly?

**Comment by diomed** [comment]
🔷 yes 🔷

**Comment by co16353sidak** [comment]
Would love to work on this @kognise if it is available for external contributors 🥇

**Comment by kognise** [comment]
@co16353sidak Thanks! Feel free to work on this :smile:

**Comment by diomed** [comment]
damn. I thought this is it, and that it's gonna be released in this version. :|

**Comment by co16353sidak** [comment]
@kognise @diomed Will work on this, and send up a PR as quickly as possible 🥇

**Comment by diomed** [comment]
@co16353sidak isn't  this just copy-pasting above code in existing css?

tho you could I guess make one for light theme.

**Comment by co16353sidak** [comment]
@diomed  basically there is a use of `scss` which is later compiled to `css` 
so this will be added to the `scss` which will then compile down to css for the same.

as for the code, more or less yes I will be starting with your suggestion above. let's see how it looks.

**Comment by co16353sidak** [comment]
@kognise

what would be more suitable here would be to design two selection colours each for dark and light mode, instead of using the darken function explicitly.

because while darken works good for light theme but for the dark theme a lighter selection colour would seem more suitable.
the addition of two selection colours can therefore allow the usage of these colour variables for each theme.

currently implementing the current scenario I have reached here:

<img width="502" alt="Screenshot 2019-04-29 at 1 09 56 AM" src="https://user-images.githubusercontent.com/35738138/56869264-0fe7f880-6a1c-11e9-9331-c129f51f3cbc.png">
<img width="563" alt="Screenshot 2019-04-29 at 1 09 52 AM" src="https://user-images.githubusercontent.com/35738138/56869265-11b1bc00-6a1c-11e9-9887-4a5c7f4cfa00.png">

these as such are not looking very nice as well.

**Comment by co16353sidak** [comment]
@kognise 
How about these:

<img width="521" alt="Screenshot 2019-04-29 at 1 25 21 AM" src="https://user-images.githubusercontent.com/35738138/56869377-a9fc7080-6a1d-11e9-93f0-729be13a768c.png">
<img width="515" alt="Screenshot 2019-04-29 at 1 20 26 AM" src="https://user-images.githubusercontent.com/35738138/56869330-0d39d300-6a1d-11e9-8dfa-0187c4379953.png">

**Comment by kognise** [comment]
@co16353sidak Those look great! I'm away from my computer right now, but I'm curious - how would the highlight look with bright colors instead of grayscale?

**Comment by diomed** [comment]
I don't like gray for white one at all

I'd go with baby blue https://www.colorhexa.com/89cff0

or lavender

**Comment by co16353sidak** [comment]
we are not really using any colour palette as such in water.css, these selection colours are as of now just darker or lighter variations of the `background-alt` colour

**Comment by kognise** [comment]
Either way I'd recommend making a `$selection` variable for the selection color. We'll have to remember to put that in the readme. 

If you've already done this, sorry. I'm only on my phone right now.

**Comment by co16353sidak** [comment]
@kognise already made a `$selection` variable, just waiting on colour finalisation from your end. 😃

**Comment by diomed** [comment]
pls dont let it be plain gray :|

**Comment by diomed** [comment]
can we get this going?

**Comment by kylejrp** [comment]
@co16353sidak Thanks for for your help on this issue! We appreciate your contribution.

@diomed Thanks for logging this issue. We're fast with releases and we appreciate any specific contributions or suggrstions you have. It's important to be patient and professional towards other contributors while this issue it's worked on - especially since this issue it's marked as a "good first issue".

@kognise Please let us know if you have any recommendations for colors here or if you're looking for suggestions.

**Comment by diomed** [comment]
he did ask for color advice for color to put on white

**Comment by kognise** [comment]
I can try some things! [This](http://www.flatuicolorpicker.com/) is also a super useful website for this kind of thing.

**Comment by co16353sidak** [comment]
@kognise @kylejrp 
Any colour we do choose, will be the standard that will be used for every selection job, so that is why I was not hasty in sending a PR right away but instead wanted to discuss the colour scheme first.
I feel that the mods here are authorised enough to take such a decision here.

**Comment by diomed** [comment]
white theme selection should have 
`$background-color:#E8EAF6`

and **color: black**
black is a must.

**Comment by diomed** [comment]
@kognise ?

**Comment by diomed** [comment]
@kognise ?

come on guys

**Comment by kylejrp** [comment]
@co16353sidak Can you make a pull request with the work you have for this? We can merge what you have so far and then we can always change the color later if necessary. I just don't want your work to get lost.

**Comment by co16353sidak** [comment]
@kylejrp 
i don’t have my laptop with me at the moment and any changes I made are yet to be pushed onto my fork. I will be able to finalise all the required changes by the end of today and then send a PR, I am sorry for this delay

**Comment by kylejrp** [comment]
@co16353sidak Oh hey sorry, no worries at all! There's absolutely no rush to get these changes in. I just wanted to make sure that we don't lose your changes.

Take your time and thanks for all of your work. 😊

**Comment by co16353sidak** [comment]
@kylejrp @kognise please review PR, what I was thinking that I log in a separate issue where in we can discuss the colours to be used and in the meanwhile merge in the basic selection variable colours I have used.
Happy to do the same.
